tests/functional/disaster-recovery/metro-dr-pc/test_failover_and_relocate_app.py

```
@@ -0,0 +1,344 @@
+        logger.info(
+            f"Wait for {wait_time} seconds before starting Relocate of applications"
+        )
+        time.sleep(wait_time)
+        secondary_cluster_name = get_current_secondary_cluster_name(
+            workloads[0].workload_namespace, constants.SUBSCRIPTION
+        )
+
+        for wl in workloads:
+            if config.RUN.get("mdr_relocate_via_ui"):
+                logger.info(
+                    f"Start the process of Relocate from ACM UI for {wl.workload_type} workload"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)